internal/pkg/enccid/enc_hamt.go

```diff
@@ -0,0 +1,111 @@
+package enccid
```

Comment:
i would move this to an `ipld` package along with the ipld store you added.

Reply:
removed